src/types/event.ts

```diff
@@ -11,6 +11,7 @@ const EventSchema = z.object({
 	rsvp: z.string(),
 	city: z.string().default(''),
 	address: z.string().nullable(),
+	image: z.string().default(''),
```

Comment:
?

Reply:
The API response also contains `image` field with the url of the img.
See example response: https://github.com/naugtur/meetjs.pl/pull/86/files#diff-2d2bd27326bd9aa0d16a847a6d08085ac3d7b56307dc4c06d3061bd6f8382058R20